src/molecules/Dialog/Dialog.tsx

```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
       children,
       width,
       actions,
-      withContentPadding = true,
+      withContentPaddingX = true,
+      withContentPaddingY = true,
       withBorders = false,
       contentMaxHeight = '60vh',
+      hideClose,
+      additionalInfo = (
+        <div>
+          <Typography bold>
+            Sign off to confirm that your approved experiences are ready for
+            publication to our partners.
+          </Typography>
+          <Typography>
+            When the selected group of experiences are ready to be publish,
+            check the last checkbox that says &#39;Selected set of experiences
+            are ready for publishing&#39;. This will notify the publisher that
+            you are ready to publish them.
+          </Typography>
+        </div>
+      ),
```

Comment:
This shouldn't be the default value of additionalInfo

Reply:
Quite right 😅 removed